kamailio/default.cfg

```diff
@@ -150,7 +150,7 @@ loadmodule "xlog.so"
 
 ####### FIFO support for Management Interface  ########
 loadmodule "mi_fifo.so"
-modparam("mi_fifo", "fifo_name", "/tmp/kamailio_fifo")
+modparam("mi_fifo", "fifo_name", "/var/run/kamailio/kamailio_fifo")
```

Comment:
Just tested and it does use the correct location if I comment out this line. Would you prefer to remove this line completely or just change it to the correct path?